proto/edgehog/device/forwarder/message.proto

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 SECO Mind Srl
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package edgehog.device.forwarder;
+
+import "edgehog/device/forwarder/http.proto";
+import "edgehog/device/forwarder/ws.proto";
+
+message Message {
+    oneof protocol {
+        Http http = 1;
+        WebSocket ws = 2;
+    }
+}
```

Comment:
```suggestion
message Message {
    bytes id = 1;

    oneof protocol {
        Http http = 2;
        WebSocket ws = 3;
    }
}
```

after irl discussions, I think it would be best to have a global `id` in the Message struct.
its meaning would be tied to the underlying protocol:
- for http, the single http request
- for ws, the currently open websocket session
- for internal messages: if we receive a message with id `i` and it makes us respond with an internal message, we will mark said message with id `i` for tracing purpose.

Reply:
Imho having a top level `id` is ambiguous because `id` evokes something unique, while it would not be unique for, e.g., `socket_id`. I would rather split this in semantically relevant ids (e.g. separate `request_id`, `socket_id`) that live inside the relevant Message, like before.